### PR TITLE
OCPBUGS-3860: Add mastersSchedulable field to AgentClusterInstall

### PR DESCRIPTION
**Pull request by zaneb**
This adds the new mastersSchedulable field in the AgentClusterInstall cluster-manifest, to allow users of the agent install method to specify a schedulable control plane at install time for clusters that have more than 4 nodes.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -189,6 +189,7 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
+	github.com/nmstate/nmstate/rust/src/go/nmstate v0.0.0-20220811151154-801022633c42 // indirect
```
Comment (r4f4) [inline go.mod:192]: It looks like `nmstate` was not vendored in:
```
 vendor/github.com/openshift/assisted-service/pkg/staticnetworkconfig/generator.go:13:2: cannot find package "github.com/openshift/installer/vendor/github.com/nmstate/nmstate/rust/src/go/nmstate" in:
	/go/src/github.com/openshift/installer/vendor/github.com/nmstate/nmstate/rust/src/go/nmstate 
```

Comment (zaneb) [reply go.mod:192]: My bad, forgot to `git add`.
I don't think this is gonna compile in CI though :(

**Comment by zaneb** [comment]
/close
Closing in favour of #7439

**Pull request closed without merging**